composer.json

```diff
@@ -7,6 +7,7 @@
     "require": {
         "php": ">=5.6.4",
         "intervention/image": "^2.4",
+        "laracasts/flash": "^3.0",
```

Comment:
why this pull request like https://github.com/AT-PHPIntership/food-market-order/pull/72/files